docs/tool-output-format.md

```
@@ -0,0 +1,29 @@
+# Tool output format
+
+LLxprt Code sends tool outputs to the model as **plain multi-line text** (not JSON).
+
+This avoids double/triple JSON-encoding that produces heavily escaped strings, which are harder for the model to read (especially for code).
+
+## Format
+
+The tool output text is formatted as:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)